src/file-4.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:30941/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:30941/project/issues?id=sample&branch=feature3&issues=AWEo2NlDtNrcb5Pj0qqD&open=AWEo2NlDtNrcb5Pj0qqD)

[ISSUE_KEY]: AWEo2NlDtNrcb5Pj0qqD